core/src/test/java/org/infinispan/remoting/transport/MockTransport.java

```diff
@@ -249,6 +250,16 @@ public boolean isMulticastCapable() {
       return true;
    }
 
+   @Override
+   public void checkCrossSiteAvailable() throws CacheConfigurationException {
+
+   }
+
+   @Override
+   public String localSiteName() {
```

Comment:
for me we only need a method (maybe I'm missing something)

Reply:
not sure what do you mean here :(